client/src/pages/Upcoming/index.tsx

```diff
@@ -325,6 +336,7 @@ const Upcoming = () => {
                                     >
                                         Submit
                                     </Button>
+                                    {jobError && <Alert sx={{mt: 1}} severity="error">All fields must be filled out</Alert>}
```

Comment:
Same issue as side nav add job ^

Reply:
Though, if we see this button, then we know there's a current cycle, maybe for this one, it can be changed to "All fields must be properly filled out"